src/tir/usmp/unified_static_memory_planner.cc

```diff
@@ -54,14 +55,62 @@ IRModule PlanMemory(const IRModule& mod, String algo) {
   PrimFunc main_func = Downcast<PrimFunc>(mod->Lookup(::tvm::runtime::symbol::tvm_module_main));
   BufferInfoAnalysis buffer_info_analysis = ExtractBufferInfo(main_func, mod);
   Array<BufferInfo> buffer_info_arr =
-      CreateArrayBufferInfo(buffer_info_analysis->buffer_info_stmts);
+      ConvertToArrayOfBufferInfo(buffer_info_analysis->buffer_info_stmts);
+
+  // All items in RO pool should have conflicts with each other in this RO pool
```

Comment:
This part should not be here.

This needs to part of ExtractBufferInfo pass.
I would prefer if we can make it part of ExtractBufferInfo pass because that analysis pass is reponsible in producing the BufferInfo nodes with correct conflicts rather than doing a correction later like this.

Please keep this part of code for integration of sub functionality of USMP.

Reply:
Moved to ExtractBufferInfo